azerbaijani/nağılşmd

```diff
@@ -0,0 +1,16 @@
+Biri var idi, biri yox idi. Bir ayı, günü keçmiş kişi var idi. Bir arvadı var idi.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.